### ISSUE TITLE
Older avatars not showing larger version of image

### ISSUE THREAD

**Issue by sudara** [sudara/alonetone#799]
It's possible we never handled this properly, but the expected behavior is that we show the largest image we have and let the browser upscale.

@manfred is working the fastly stuff shortly, so maybe this will be rendered moot — as long as we can upscale images or deliver the largest size we have.

Example: 

https://alonetone.com/fabiokeiner

https://cdn.alonetone.com/pics/53996/LOOK_large.jpg

<img width="542" alt="image" src="https://user-images.githubusercontent.com/472/65837857-915df280-e2fc-11e9-9d10-766e596da9d7.png">

**Comment by Manfred** [comment]
Yes, I think they should come back when we deploy the Active Storage stuff. Otherwise I will figure out how to migrate other sizes.

**Comment by sudara** [comment]
Fuck yes.
